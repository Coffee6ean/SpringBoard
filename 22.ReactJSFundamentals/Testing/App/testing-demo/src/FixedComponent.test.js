import React from 'react';
import { render } from '@testing-library/react';
import FixedComponent from './FixedComponent';

test('it renders without crashing', () => {
    render(<FixedComponent/>)
});

test('if matches snapshot', () => {
    const { asFragment } = render(<FixedComponent/>);
    expect(asFragment()).toMatchSnapshot();
})
