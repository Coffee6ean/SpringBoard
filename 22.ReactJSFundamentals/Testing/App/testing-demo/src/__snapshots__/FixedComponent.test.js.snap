// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`if matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <p>
      Hello, Im a FixedComponent!
    </p>
    <p>
      Here are some numbers:
    </p>
    <p>
      42
    </p>
    <p>
      43
    </p>
    <p>
      44
    </p>
  </div>
</DocumentFragment>
`;
